index.html

```diff
@@ -10,120 +10,120 @@
 </head>
 
 <body>
-  <div class='meni'>
-    <button id='prodaja' class='meniBtn'>Prodaja</button>
-    <button id='nabavka' class='meniBtn'>Nabavka</button>
-    <button id='nivelacija' class='meniBtn'>Nivelacija</button>
-    <button id='pregledi' class='meniBtn'>Pregledi</button>
-    <button id='aplikacija' class='meniBtn'>Aplikacija</button>
+  <div class='menu'>
+    <button class='menuBtn' onclick="styleMenuNav(0);sell()" >Prodaja</button>
+    <button class='menuBtn' onclick="styleMenuNav(1);importArticle()">Nabavka</button>
+    <button class='menuBtn' onclick="styleMenuNav(2);nivelation()">Nivelacija</button>
+    <button class='menuBtn' onclick="styleMenuNav(3);show()">Pregledi</button>
+    <button class='menuBtn' onclick="styleMenuNav(4);applicationClick()">Aplikacija</button>
```

Comment:
Ne mora da se na svako dugme postavi `id`, pa kasnije da se cupa element na osnovu id-a.  
Moze sve preko `onclick` da se definise, zahteva manje kod, i lakse se posle modifikuje.

Reply:
da. ovo je vec bilo glupo sa moje strane ali mi je to bio najmanje bitan deo koda. Necete valjda da mi smanjite ocenu zbog toga?